helpdesk_mgmt/data/helpdesk_data.xml

```diff
@@ -320,7 +320,7 @@
         <record id="mail_alias_lead_info_helpdesk" model="mail.alias">
             <field name="alias_name">help</field>
             <field name="alias_model_id" ref="model_helpdesk_ticket" />
-            <field name="alias_user_id" ref="base.user_admin" />
+<!--            <field name="alias_user_id" ref="base.user_admin" />-->
```

Comment:
Reason for leaving this commented? why not remove entirely if not working or fixing it with the correct record?